tensorflow/compiler/tests/xla_call_module_test.py

```diff
@@ -496,7 +497,7 @@ def f(x):
     }
   }
 """
-
+  '''
```

Comment:
Could you use ```test.is_built_with_rocm()``` instead of just commented out in the code? Also, in your tracking ticket https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/6264 could you also add the new added commit for tracking? Thanks